### PR TITLE
doc: remove references to JIRA

### PR DESCRIPTION
**Pull request by dbkinder**
Replace references to JIRA with github issue tracker

[skip ci]

Signed-off-by: David Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by kad** [comment]
👍

File: doc/quick_start/contributor-guide.rst [hunk 1]
```diff
@@ -42,15 +42,14 @@ As a Linux developer, you should also be familiar with common development tools
...
 =====================================
 
-Before starting on a patch, first check in `Ostro Project JIRA`_ to see what’s been reported on the issue you’re addressing. 
+Before starting on a patch, first check in `Ostro Project Issues`_ to see what’s been reported on the issue you’re addressing. 
 Have a conversation on the mailing list to see what others think of your issue (and proposed solution). 
 You may find others that have encountered the issue you're finding, or with similar ideas for changes or additions. 
 If you don't, send a message to the mailing list to introduce and discuss your idea with the development community.
 
-After logging into JIRA, click “Issues” to search for open items. To avoid duplication, it's always a good practice 
+After access the ``ostro-os`` GitHub repo, click “Issues” to search for open (and closed) items. To avoid duplication, it's always a good practice
```
Comment (gvancuts) [inline doc/quick_start/contributor-guide.rst]: Shouldn't it say "After accessing..." (instead of access) to be grammatically correct?

**Comment by gvancuts** [comment]
Just one question from me on grammar but otherwise :+1: !

**Comment by dbkinder** [comment]
fixed geoffroy's comment... ready to merge

**Comment by gvancuts** [comment]
:+1:

**Pull request merged**